selectinf/randomized/posterior_inference.py

```diff
@@ -31,39 +31,46 @@ def __init__(self,
                  query,
                  observed_target,
                  cov_target,
-                 cov_target_score,
+                 regress_target_score,
+                 dispersion,
                  prior,
-                 dispersion=1,
```

Comment:
Don't think this dispersion argument should be here. Dispersion could be per-query whereas posterior is for a target....